### DISCUSSION TITLE
Right click on destkop to open menu

### DISCUSSION THREAD

**Discussion by resucutie** [Schneegans/Fly-Pie#274] [Ideas & Feature Discussions]
Instead of relying on a key combination or Super + RClick, right clicking the desktop could bring the menu. On pure GNOME, that is, without extensions, the right click menu is with not much of a use, only having options related to display and wallpapaer, so adding Rclick functionality in there would be very appreciated

**Comment by Schneegans** [comment]
Hi there! This is actually a good idea! I will not have the time to work on this in the near future, but maybe some can implement this. It should be pretty similar to the <kbd>Super</kbd>+<kbd>RMB</kbd> functionality which is already there.

**Comment by Schneegans** [comment]
I created #280 accordingly.

**Reply by resucutie** [reply]
Thanks! I appreciate your work
